src/pynxtools_xps/vms/casa_data_model.py

```diff
@@ -209,31 +254,28 @@ def _split_line(self, line):
         return [re.sub(r"[\(*?\*)]", "", text) for text in result]
 
 
-def map_energy_type(energy_str):
-    """Map energy names to NXDL-compliant concepts."""
-    replacements = {
-        "BE": "binding",
-        "KE": "kinetic",
-        "binding energy": "binding",
-        "binding": "kinetic",
-        "Binding": "binding",
-        "Kinetic": "kinetic",
-    }
-    return replacements.get(energy_str)
-
-
 @dataclass
-class CasaAlignment(XpsDataclass):
-    """An object to store one CasaXPS alignment."""
+class CasaEnergyCalibration(XpsDataclass):
+    """An object to store one CasaXPS energy calibration."""
 
     energy_type: str = "binding"
     energy_offset: float = 0.0
     energy_offset_units: str = "eV"
-    measured_energy: float = 0.0
-    measured_energy_units: str = "eV"
+    measured_energies: list = field(default_factory=list)
+    measured_energies_units: str = "eV"
     aligned_energy: float = 0.0
     aligned_energy_units: str = "eV"
     operation: str = "eV"
+    range_calibration: bool = False
+
+
+@dataclass
+class CasaIntensityCalibration(XpsDataclass):
+    """An object to store one CasaXPS energy calibration."""
+
+    ## TODO ##
+    power_law_coefficient: float = 0.0
+    tf_function_ordinate_no: int = 0
```

Comment:
Probably you forgot to remove the `TODO` part, as it does not describe anything.